mne/io/ctf/ctf.py

```diff
@@ -137,8 +137,10 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             fnames.append(meg4_name)
             last_samps.append(sample_info['n_samp'] - 1)
             raw_extras.append(sample_info)
+            first_samps = [0, ] * len(last_samps)
```

Comment:
FYI you can just do `[0] * len(last_samps)` but it's okay even without that change